### ISSUE TITLE
bug disguise after pull origin master 676c248

### ISSUE THREAD

**Issue by dzurrahman** [rathena/rathena#8001]
### rAthena Hash

676c2483a

### Client Date

2022-03-30

### Server Mode

Renewal

### Result

disguise event can be anything answer is correct

![image](https://github.com/rathena/rathena/assets/56845420/f0f00264-5dc6-4c7a-87c9-d9c3a5940fa5)
![image](https://github.com/rathena/rathena/assets/56845420/d9c0f8dd-efe5-4d6e-83b6-f47dfed01021)

### Expected Result

should incorrect

### How to Reproduce

active disguise -> say anything

### Official Information

-

**Comment by datawulf** [comment]
Does your disguise script use "--ja--" as monster name? Change that to "--en--" if you are not using JapaneseName in database.

**Comment by dzurrahman** [comment]
--ja--, --en--- ? both of them not avaiable in disguise.txt code, 

but when i rollback commit it works

![image](https://github.com/rathena/rathena/assets/56845420/69ddb688-41ab-40f3-9661-ed1debbc98e9)

there is no changes in disguise.txt script

**Comment by NexoScript** [comment]
This problem started since they uploaded this to the master branch
https://github.com/rathena/rathena/commit/2487d2bf790e19fe41a0926d25fa3887b79fbaf3

**Comment by Mictlah** [comment]
Yeah, at moment to make @monster command, the monster don't have a name.

**Comment by datawulf** [comment]
Being it intended or not, but rathena as well as many scripts use Japanese name and defaulting to English name when it's empty has been removed with one of the latest changes.
